_posts/2024-08-01-contributing-to-che-docs.adoc

```diff
@@ -0,0 +1,51 @@
+---
+title: Contributing to Eclipse Che documentation
+layout: post
+author: Jana Vrbkova
+description: >-
+  Contribute to Eclipse Che documentation using an Eclipse Che workspace.
+categories: []
+keywords: ['che-docs', 'documentation', 'write', 'contribute']
+slug: /@deerskindoll/contributing-to-che-docs
+---
+
+= Contributing to Eclipse Che documentation
+
+Writing about Eclipse Che is even easier than you think.
+The link:https://github.com/eclipse-che/che-docs[Eclipse Che documentation repository on GitHub] was created with community contribution in mind,
+featuring tools you need to create and test your content.
+And from the initial draft to submitting a PR,
+you can do it all in the environment you're already familiar with: an Eclipse Che workspace.
+
+.Getting started
+
+Use either link:https://eclipse.dev/che/docs/stable/administration-guide/installing-che/[your already installed Eclipse Che]
+or click the Developer Workspace button directly in the che-docs repository
+to directly open a clone of the repository in an Eclipse Che workspace hosted
+by Red Hat in the the link:https://developers.redhat.com/developer-sandbox?source=sso[Developer Sandbox].
+
+*add the button?*
+
+*describe plugins?*
+
+*vale sync command*
+
+The Vale extension included in the che-docs repository follows the grammatical,
+spelling and stylistic rules from the link:https://redhat-documentation.github.io/supplementary-style-guide/[Red Hat supplementary style guide].
+Before you start writing,
+read the section about link:https://redhat-documentation.github.io/supplementary-style-guide/#technical-examples[documenting commands].
+
+.Writing tips
+
+A good technical document is easy to read and easy to follow.
```

Comment:
```suggestion
A good technical document is easy to read, easy to follow, and now easy to contribute!
```

Reply:
not ideal suggestion (feel free to skip it) but somewhere in the article we should articulate that the game-changer is contributing and spinning up an environment for contribution with one click